### ISSUE TITLE
Add STL Model of the build plate [enhancement]

### ISSUE THREAD

**Issue by lthown** [DrLex0/FFCP-GCodeSnippets#22]
There's a model of the FFCP build plate on https://www.thingiverse.com/thing:3772642

This file can be loaded into the PrusaSlicer using the "bed shape" setting

<img width="464" alt="2022-09-26 18_45_02-Settings" src="https://user-images.githubusercontent.com/750471/192405774-e1286dcd-38d3-4bec-a6b7-257b6638e084.png">

The results are quite nice, though the file appears to be too far back by a few mm

<img width="1096" alt="2022-09-26 18_46_48-_Untitled - PrusaSlicer-2 5 0 based on Slic3r" src="https://user-images.githubusercontent.com/750471/192405955-2d44aa14-62a6-4da7-afaf-09e7661b5c45.png">

Still a nice way to visualize the print within the printer.

**Comment by DrLex0** [comment]
It's not entirely useless, because it adds some visual reference points. and this avoids the risk of placing things upside down, which I sometimes end up doing. I'll see if I can tweak the model to be 100% correct and maybe also add a texture.

**Comment by lthown** [comment]
cool beans

**Comment by DrLex0** [comment]
Added, but as you'll see, because PS relies on absolute paths to these files, it is not possible to load the model (and texture) fully automatically while importing the ini.
I have added functionality to the config helper page that can avoid having to manually configure the files for each of the printer profiles, although in your case this is probably a better option than going through the whole process again.

This would all be simpler (at least for users) if I could distribute these configs as a vendor preset in PrusaSlicer, but this is blocked by their unwillingness to solve the issue with the tool number.
